setup.py

```diff
@@ -21,9 +21,9 @@
 
```

Comment:
We need to freeze this, but not harder than upstream.
They will get an update "soon" anyway, so we will get back to this.
Also, we should at least try to leave werkzeug (did you have problems with it?)